addon/helpers/ember-sparkles/arc-translate.js

```diff
@@ -0,0 +1,16 @@
+import Ember from 'ember';
+
+export function emberSparklesArcTranslate([ centroid ], { height, width }) {
+  return function(d) {
+    let x, y;
+    [x, y] = centroid(d)
+    x = x + width;
+    y = y + height;
+    let coordinates = [];
+    coordinates.push(x);
+    coordinates.push(y);
+    return `translate(${coordinates})`;
```

Comment:
let's talk about this part - you're mutating some variables, and we don't need to use the array.